clients/privacy-center/__tests__/common/geolocation.test.ts

```diff
@@ -56,6 +70,17 @@ describe("getGeolocation", () => {
       expect(geolocation).toBeNull();
     });
 
+    it("ignores invalid three-character country geolocation headers", async () => {
+      const req = createRequest({
+        url: "https://privacy.example.com/fides.js",
+        headers: {
+          "CloudFront-Viewer-Country": "USA",
```

Comment:
Would CloudFront ever return a non-valid code like this? If so do we care to handle it differently rather than just ignore it? If not, is the test giving a false impression?

Reply:
From AWS: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/adding-cloudfront-headers.html#cloudfront-headers-viewer-location
> CloudFront-Viewer-Country – Contains the two-letter country code for the viewer's country

I'd prefer to see a warning/error log if something unexpected happened at runtime, but handle this by defensively ignoring the invalid input... We created PROD-2035 for better error logs: https://ethyca.atlassian.net/browse/PROD-2035